build/crd-samples/devices/CC2650-device-model.yaml

```diff
@@ -57,8 +57,8 @@ spec:
     bluetooth:
      characteristicUUID: f000aa0204514000b000000000000000
      dataWrite:
-      ON: [1]
-      OFF: [0]
+      "ON": [1]
```

Comment:
Sorry I just forget what does the [1] [2] [3] represent? So I think it's worth giving some comments here and below?

Reply:
+1

Reply:
Here [1] , [2], [3] etc represents the []byte value to be written into the device for the corresponding key value of the property. Have given a brief explanation in the comment. Thanks